ledger-api/grpc-definitions/com/daml/ledger/api/v1/version_service.proto

```diff
@@ -57,7 +57,9 @@ message FeaturesDescriptor {
 
 }
 
-// Whether the Ledger API server provides the user management service.
 message UserManagementFeature {
+  // Whether the Ledger API server provides the user management service.
   bool supported = 1;
+  // The maximum number of rights that can be assigned to a single user.
```

Comment:
Please say what happens when the value is not set or set to zero. I can see that the ledger-api-test-tool assumes no limit in this case.

Reply:
Ok. Added an explicit note about `0` value and changed to an unsigned type.